arrow-libs/fx/arrow-fx-coroutines/src/jvmTest/kotlin/examples/example-schedule-02.kt

```diff
@@ -1,13 +1,13 @@
 // This file was automatically generated from Schedule.kt by Knit tool. Do not edit.
 package arrow.fx.coroutines.examples.exampleSchedule02
 
-import kotlin.time.seconds
-import kotlin.time.milliseconds
 import kotlin.time.ExperimentalTime
 import arrow.fx.coroutines.*
+import kotlin.time.Duration.Companion.milliseconds
+import kotlin.time.Duration.Companion.seconds
 
 @ExperimentalTime
 fun <A> complexPolicy(): Schedule<A, List<A>> =
-  Schedule.exponential<A>(10.milliseconds).whileOutput { it.inNanoseconds < 60.seconds.inNanoseconds }
+  Schedule.exponential<A>(10.milliseconds).whileOutput { it < 60.seconds }
```

Comment:
I've simplified the Schedule example. Duration implements `Comparable`, I reckon we don't need to convert it to nanoseconds.